.eslintrc

```diff
@@ -9,5 +9,6 @@
   "rules": {
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
+    "react/no-did-mount-set-state": 0, // necessary for server-rendering
```

Comment:
I can, but it should be a universal exception, because it's necessary for *any* component. I need to disable this rule upstream in the airbnb config too - `setState` in `componentDidMount` is a good thing.